nf_core/pipeline-template/.github/workflows/nf-test.yml

```diff
@@ -0,0 +1,141 @@
+name: Run nf-test
+on:
+  push:
+    branches:
+      # https://docs.renovatebot.com/key-concepts/automerge/#branch-vs-pr-automerging
+      - "renovate/**" # branches Renovate creates
+  pull_request:
+    branches: [dev]
```

Comment:
I think we should also run the tests on PRs to master, and probably any PR actually, not only to dev.

Reply:
was just copying from `ci.yml`

Reply:
```suggestion
```

Is this what you were suggesting @mirpedrol

Reply:
yes 👍